body_test.go

```diff
@@ -16,6 +16,10 @@ func setup(t *testing.T) (*os.File, []byte, func()) {
 	var b bytes.Buffer
 	var err error
 
+	if err = os.MkdirAll("testdata", os.FileMode(0755)); err != nil {
```

Comment:
nit: the explicit cast isn't required as it can be inferred by the compiler, e.g. you could just write:

```
if err = os.MkdirAll("testdata", 0755); err != nil {
```

Reply:
Cool, I assumed it wouldn't know how to do that!